README.md

```diff
@@ -1,3 +1,41 @@
-# orca-shipnoise
-This is the front end of shipnoise.net!!
+# ShipNoise.Net
 
+This repo specifies the web backend and frontend for the ShipNoise prototype site, a way to visualize the noisiest ships in the Salish Sea.
+
+The demo site can be found here: https://main.d27wklsx9a9q36.amplifyapp.com/
+
+## Contributing
+
+Please check out the CONTRIBUTING doc for tips on making a successful contribution.
+
+## How it works
+
+At this stage, this repo is merely a prototype for the real ShipNoise.Net, a proof-of-concept for future development. 
+It is automatically deployed to the AWS Amplify environment from this repo's main branch. Pushes to the main branch trigger a new deployment.
+
+The demo site is available at: https://main.d27wklsx9a9q36.amplifyapp.com/ 
```

Comment:
This link is given just above, so probably doesn't need to be repeated here... 
(This is mostly Scott just practicing leaving a single-line comment.)